.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
         - CONDA_PY=27  CONDA_NPY=19
 
 before_install:
-    - rm -rf nco nco-bindings pyaxiom
+    - rm -rf nco nco-bindings
```

Comment:
@rsignell-usgs now we have `pyaxiom` for Mac (and Windows) sorry for the misleading false positive in PR #229 due to my hackish way to avoid the build on travis.  This PR is safe to be merged.